setup.cfg

```diff
@@ -50,6 +50,9 @@ test =
     pytest
     pytest-cov
     pytest-mock
+typing =
+    mypy==0.782
```

Comment:
Why pin the dependency?

Reply:
Because mypy is getting better with every release. We don't want the CI to break just because a new release was done, and now the checks are better and mypy detected a new bug in our app. We'll periodically upgrade.

Reply:
I would definitely forget to bump, but if you got that covered :+1:

Reply:
Yeah 👍